cdap-ui/app/hydrator/controllers/create/partials/pipelineupgrade-modal-ctrl.js

```diff
@@ -15,7 +15,7 @@
  */
 
 angular.module(PKG.name + '.feature.hydrator')
-  .controller('PipelineUpgradeModalController', function ($scope, rPipelineConfig, HydratorUpgradeService, $rootScope, HydratorPlusPlusConfigStore, $state, DAGPlusPlusFactory, GLOBALS, HydratorPlusPlusLeftPanelStore) {
+  .controller('PipelineUpgradeModalController', function ($scope, rPipelineConfig, HydratorUpgradeService, $rootScope, HydratorPlusPlusConfigStore, $state, DAGPlusPlusFactory, GLOBALS, HydratorPlusPlusLeftPanelStore,rIsImport) {
```

Comment:
nit: Could we have a blank line like this? `, rIsImport) {`

Reply:
Fixed.